fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -160,12 +164,22 @@ public List<Column> listPartitionColumns(String databaseName, String tableName,
     public List<PartitionInfo> getPartitions(Table table, List<String> partitionNames) {
         try (Connection connection = getConnection()) {
             List<Partition> partitions = schemaResolver.getPartitions(connection, table);
+            String maxInt = IntLiteral.createMaxValue(Type.INT).getStringValue();
```

Comment:
what if long type?

Reply:
The incoming partitionNames collection defined by the interface hierarchy is of type string, and the use of string instead of long in this area is for the convenience of using contains in the next comparison step.